themes/bootstrap5/js/account_ajax.js

```diff
@@ -23,15 +23,42 @@ VuFind.register('account', function Account() {
 
   var _submodules = [];
   var _clearCaches = false;
-  var _sessionDataPrefix = "vf-account-status-";
+  var _sessionDataPrefix = "vf-account-status-2-";
```

Comment:
Is this 2 intentional for forward-compatibility, or is it just an artifact of your own testing/development? If it's meant to be used long-term, maybe we can change it in a less cryptic way, like `vf-acct-status-data-` or something like that which makes sense on its own without knowing about historical iterations.

Reply:
It's just a version number just in case we ever have version 3, but since it's unlikely, I can change it.

Reply:
I actually think the new code is probably sufficiently backward-compatible anyway (if you have old session data with the new code, it will carry forward some useless junk, but I don't think it will break anything... and the junk data won't outlive your browser session, so it's not a big deal).

Reply:
True, I reverted the change.